modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/SampleData_vertebrates_conf.pm

```diff
@@ -0,0 +1,55 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+Please email comments or questions to the public Ensembl
+developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+Questions may also be sent to the Ensembl help desk at
+<http://www.ensembl.org/Help/Contact>.
+
+=head1 NAME
+
+Bio::EnsEMBL::Production::Pipeline::PipeConfig::SampleData_vertebrates_conf
+
+=head1 SYNOPSIS
+
+
+=head1 DESCRIPTION
+
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::SampleData_vertebrates_conf;
+
+use strict;
+use warnings;
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::SampleData_conf');
+
+sub default_options {
+    my ($self) = @_;
+
+    return {
+        # inherit other stuff from the base class
+        %{ $self->SUPER::default_options() },
+	    # species with curated sample genes
+        antispecies   => ['homo_sapiens'],
```

Comment:
We might need more species here like mouse, rat... @james-monkeyshines do you know the full list or should we check with the genebuild team?

Reply:
Yes, agree about mouse and rat, probably zebrafish too... Need to get genebuild to give us a full ist too. Or maybe it would be better to do this only for new species, not updated ones. (If an geneset update removed the sample gene a datacheck would fail, so that would be flagged up before handover to us.)

Reply:
@leannehaggerty: Could you please give us a list of Vertebrates species for which we don't want to overwrite the sample data?

Reply:
Sorry I missed this. These are my thoughts, but happy to provide a list if you think necessary:
I think James' suggestion to run for all new species is probably the best solution - that way everything that already exists will stay intact. 
In the event of a gene set update we (Genebuild) will need to check before handover that the sample gene still exists in the gene set.
And finally if there is a specific sample gene that needs to be set for a species in the future, i.e. we don't want to use the sample gene script to update it, then we should add this species to the 'antispecies' list.

Reply:
Hi all

Have discussed with the Genebuilders and think the initial list should be human, mouse, zebrafish, pig, sheep, horse and cow. 
However, for the sheep that has just been handed over for e101 (sheep (rambouillet)), the script will need to be run, just this once. 
Does this sound ok? 

When new species need to be added to the list would you prefer us to update SampleData_vertebrates_conf.pm via a PR or is there a static list we should add to?

Cheers